requirements.txt

```diff
@@ -91,7 +91,7 @@ pynacl==1.5.0
     # via paramiko
 python-on-whales==0.70.1
     # via -r requirements.in
-pyyaml==6.0
+pyyaml==6.0.1
```

Comment:
This needs to be changed in the `requirements.in` file, otherwise this will get overwritten next time this file is generated. I'll create a PR for this.

Reply:
The requirement is loose (>=6.0), so it will work without any changes. Changing it would be technically *more* correct though. We probably also want to add python 3.12 to the build matrix at the same time.

Reply:
I changed the .in file [here](https://github.com/adobe/buildrunner/pull/144).. I didn't touch adding 3.12 though.

Reply:
I guess that is correct.